src/components/Shared/MessageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import getAvatar from '@lib/getAvatar';
+import type { FC } from 'react';
+import { useState } from 'react';
+import { useAppStore } from 'src/store/app';
+
+import Follow from './Follow';
+
+const MessageHeader: FC = () => {
+  const [following, setFollowing] = useState(false);
```

Comment:
Seems like this needs to get the initial following state from the API. The user may be already followed prior to loading the component.

Reply:
Yes, was waiting for Elise's PR to get merged, this is all sorted now